### PR TITLE
Add Falcon to the ASGI framework list

### PR DESCRIPTION
**Pull request by vytas7**
Feel free to reject this one depending on the policy... But I saw that our friends at Blacksheep managed to add theirs, so thought giving it a shot :smiling_imp:
(Falcon is otherwise already included in @florimondmanca 's Awesome ASGI.)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -488,6 +488,9 @@ You write your API function parameters with Python 3.6+ type declarations and ge
 
 Its most distinctive features are built-in support for dependency injection, automatic binding of parameters by request handler's type annotations, and automatic generation of OpenAPI documentation and Swagger UI.
 
+### Falcon
+[Falcon](https://falconframework.org) is a minimalist REST and app backend framework for Python, with a focus on reliability, correctness, and performance at scale.
```
Comment (Kludex) [inline docs/index.md:493]:
```suggestion

[Falcon](https://falconframework.org) is a minimalist REST and app backend framework for Python, with a focus on reliability, correctness, and performance at scale.
```

**Review by Kludex — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @vytas7 ! :)

**Comment by vytas7** [comment]
Thanks for merging this @Kludex! :shipit: 
I just want to add that Falcon has extensive functional testing with Uvicorn in CI for ASGI on all supported Python versions, and even one gate for Uvicorn's WSGI mode; there are also E2E tests verifying WebSocket and SSE in two browsers.

**Comment by Kludex** [comment]
> Thanks for merging this @Kludex! :shipit: I just want to add that Falcon has extensive functional testing with Uvicorn in CI for ASGI on all supported Python versions, and even one gate for Uvicorn's WSGI mode; there are also E2E tests verifying WebSocket and SSE in two browsers.

Cool! :rocket:

**Comment by lovelydinosaur** [comment]
Fantastic! Falcon's a great project - ace to see it listed here.

**Pull request merged**